EnoCore/Configuration/JsonConfiguration.cs

```diff
@@ -1,259 +1,165 @@
-﻿namespace EnoCore.Configuration

```

Comment:
Why does github display this as a diff? Did the linefeeds change?

Reply:
Seems like Visual Studio loves CRLF

Reply:
but for me it shows up as LF..

Reply:
i think it might be possible to enforce one kind of line endings project-wide, at least i read about that somewhere

Reply:
Otherwise it will depend on the settings of the one editing it, which is bad imho.

Reply:
but with `.gitattributes` I didn't see any changes

Reply:
There is e.g. .editorconfig, but with VS it was a clusterfuck a few years back - if you specify \n linefeeds then new classes will have \r\n boilerplate code, and any linefeeds you enter will be \n only

Reply:
as I worked with the other projects I would say that the file probably was CRLF and now is LF only.

Reply:
we can run a formater like https://restyled.io/ on all project if you like. I can set it up but you need to allow the application.